frontend/src/metabase/admin/performance/components/StrategyEditorForDatabases.tsx

```diff
@@ -0,0 +1,328 @@
+    ]);
+
+    const showLoadingSpinner = useDelayedLoadingSpinner();
+    const error = databasesResult.error || configsResult.error;
+    const loading = databasesResult.isLoading || configsResult.loading;
+
+    if (error || loading) {
+      return showLoadingSpinner ? (
+        <LoadingAndErrorWrapper error={error} loading={loading} />
+      ) : (
+        <></>
+      );
+    }
+
+    return (
+      <TabWrapper role="region" aria-label="Data caching settings">
+        <Stack spacing="xl" lh="1.5rem" maw="32rem" mb="1.5rem">
+          <aside>{PLUGIN_CACHING.explanation}</aside>
+        </Stack>
+        {confirmationModal}
+        <Box
+          style={{
+            display: "grid",
+            gridTemplateColumns: "minmax(5rem, 30rem) minmax(5rem, auto)",
+            overflow: "hidden",
+          }}
+          w="100%"
+          mx="0"
+          mb="1rem"
+        >
+          {shouldShowStrategyFormLaunchers && (
+            <Panel role="group" style={{ backgroundColor: color("bg-light") }}>
+              <Box
+                p="lg"
+                style={{ borderBottom: `1px solid ${color("border")}` }}
+              >
+                <StrategyFormLauncher
+                  forId={rootId}
+                  title={t`Default policy`}
+                  configs={configs}
+                  targetId={targetId}
+                  safelyUpdateTargetId={safelyUpdateTargetId}
+                  isFormDirty={isStrategyFormDirty}
+                />
+              </Box>
+              <Stack p="lg" spacing="md">
+                {databases?.map(db => (
+                  <StrategyFormLauncher
+                    forId={db.id}
+                    title={db.name}
+                    configs={configs}
+                    targetId={targetId}
+                    safelyUpdateTargetId={safelyUpdateTargetId}
+                    isFormDirty={isStrategyFormDirty}
+                    key={`database_${db.id}`}
+                  />
+                ))}
+              </Stack>
+              {shouldShowResetButton && (
+                <ResetAllToDefaultButton
+                  configs={configs}
+                  setConfigs={setConfigs}
+                />
+              )}
+            </Panel>
+          )}
+          <Panel
+            ref={formPanelRef}
+            hasVerticalScrollbar={formPanelHasVerticalScrollbar}
+          >
+            {showStrategyForm && (
+              <StrategyForm
+                targetId={targetId}
+                setIsDirty={setIsStrategyFormDirty}
+                saveStrategy={saveStrategy}
+                savedStrategy={savedStrategy}
+              />
+            )}
+          </Panel>
+        </Box>
+      </TabWrapper>
+    );
+  },
+);
+
+const getFieldsForStrategyType = (strategyType: StrategyType) => {
+  const strategy = Strategies[strategyType];
+  const validationSchemaDescription =
+    strategy.validateWith.describe() as SchemaObjectDescription;
```

Comment:
shouldn't SchemaObjectDescription be the return type for this method?

Reply:
Right now that call to `describe` returns a `SchemaDescription`. Maybe this indicates I misconfigured something?